go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/mohamednizar/gitstrap
```

Comment:
@mohamednizar this should be `github.com/g4s8/gitstrap`

Reply:
I have changed the module from `github.com/mohamednizar/gitstrap` to `github.com/g4s8/gitstrap`. I thoug it should   be the master repo.